main.tf

```
@@ -40,6 +40,8 @@ resource "aws_lambda_function" "logs_to_datadog" {
       DD_API_KEY_SECRET_ARN = aws_secretsmanager_secret.api-key.arn
       DD_SITE               = var.dd_site
       DD_ENHANCED_METRICS   = var.enhanced_metrics
+      ## Filter out lambda platform logs
+      EXCLUDE_AT_MATCH = "\"(START|END|REPORT) RequestId:\\s"
```

Are we sure we need starting quote here?  Doesn't seem correct to me.  Are we seeing that in documentation somewhere?

```suggestion
      EXCLUDE_AT_MATCH = "(START|END|REPORT) RequestId:\\s"
```

---

According to the documentation example it says it is needed I guess 🤷 
https://github.com/DataDog/datadog-serverless-functions/tree/master/aws/logs_monitoring#log-filtering-optional

>Include (or exclude) Lambda platform logs: "(START|END) RequestId:\s. Note: The preceding " is needed to match the start of the log message, which is in a json blob ({"message": "START RequestId...."}). Datadog recommends keeping the REPORT logs, as they are used to populate the invocations list in the serverless function views.

---

Are we sure we want to remove the REPORT ones given the last sentence there?

```Datadog recommends keeping the REPORT logs, as they are used to populate the invocations list in the serverless function views.```

---

Guessing that is probably [this](https://app.datadoghq.com/functions?cloud=aws&panel_end=1666386867202&panel_paused=true&panel_start=1666383267202&selection=aws-lambda-functions%2Blogs_to_datadog%2Bus-east-1%2B082947815797&start=1666383267202&end=1666386867202&paused=true)

---

Yeah I saw that, it depends on if we really ever plan on using that functionality in DataDog, im assuming we currently don't because we have already been excluding those logs 😅

---

If we wanted we could just continue to send the REPORT logs and exclude them from our index like we have been?

---

Yeah, let's continue to include REPORT from this function and exclude them from the index like we currently are.